static/src/javascripts/projects/common/modules/experiments/low-friction-participation.js

```diff
@@ -185,9 +185,41 @@ define([
         mediator.emit('modules:participation:clicked');
     }
 
+    function itemHovered (e) {
+        var itemLength;
+        var $lowFricButtons;
+
+        fastdomPromise.read(function() {
+            itemLength = e.currentTarget.getAttribute('data-item-id');
+            $lowFricButtons = $('.js-participation-low-fric--button');
+        });
+
+        fastdomPromise.write(function() {
+            $lowFricButtons.removeClass('participation-low-fric--button__is-highlighted');
+
+            if (itemLength > -1) {
```

Comment:
feel like you might have a race condition here since (afaik) the fastdom `read` isn't guaranteed to finish before the `write`